Client/Application/AppDelegate.swift

```diff
@@ -232,7 +225,6 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
         handleBackgroundEvent()
         TelemetryWrapper.recordEvent(category: .action, method: .background, object: .app)
-        TabsQuantityTelemetry.trackTabsQuantity(tabManager: tabManager)
```

Comment:
Note: removal of this was intentional, it has been moved into `BrowserViewController.swift`